cache-caffeine/src/test/groovy/io/micronaut/cache/SyncCacheSpec.groovy

```diff
@@ -346,12 +347,13 @@ class SyncCacheSpec extends Specification {
 
         @Cacheable
         @SingleResult
-        Flux<Integer> fluxValue(String name) {
+        Publisher<Integer> fluxValue(String name) {
```

Comment:
There is currently a bit of duplication. Since `fluxValue` and `monoValue` due the same.